index.html

```diff
@@ -3,47 +3,434 @@
+  {"SOC":"53-0000","Category":"Major","Group_Code":"53-0000","Group":"Trans","Group_Name":"Transportation and Material Moving","Occupation_name":"Transportation and material moving occupations","Total_Workers":6.953,"Median_Earn":"$646 ","Men_Earn":"$679 ","Women_Earn":"$494 ","ME":679,"WE":494,"Women_Percent":0.727540501},
+  {"SOC":"53-3020","Category":"Minor","Group_Code":"53-0000","Group":"Trans","Group_Name":"Transportation and Material Moving","Occupation_name":"Bus drivers","Total_Workers":323,"Median_Earn":"$615 ","Men_Earn":"$681 ","Women_Earn":"$572 ","ME":681,"WE":572,"Women_Percent":0.839941263},
+  {"SOC":"53-3030","Category":"Minor","Group_Code":"53-0000","Group":"Trans","Group_Name":"Transportation and Material Moving","Occupation_name":"Driver/sales workers and truck drivers","Total_Workers":2.687,"Median_Earn":"$747 ","Men_Earn":"$751 ","Women_Earn":"$632 ","ME":751,"WE":632,"Women_Percent":0.841544607},
+  {"SOC":"53-7062","Category":"Minor","Group_Code":"53-0000","Group":"Trans","Group_Name":"Transportation and Material Moving","Occupation_name":"Laborers and freight, stock, and material movers, hand","Total_Workers":1.433,"Median_Earn":"$526 ","Men_Earn":"$547 ","Women_Earn":"$455 ","ME":547,"WE":455,"Women_Percent":0.831809872},
+  {"SOC":"53-7064","Category":"Minor","Group_Code":"53-0000","Group":"Trans","Group_Name":"Transportation and Material Moving","Occupation_name":"Packers and packagers, hand","Total_Workers":385,"Median_Earn":"$438 ","Men_Earn":"$462 ","Women_Earn":"$424 ","ME":462,"WE":424,"Women_Percent":0.917748918}
+];
+
+var options = {
+    sorted: false,
+    filtered: false,
+    page: 0
+  };
+
+function updateTable() {
+    // Make a copy of the data
+    var data = occupations.slice();
+
+    // Sort occupations by women's earnings as a percent of men's
+    if (options.sorted) {
+      if (options.sorted === 'asc') {
+        data.sort(function (a, b) { return a.Women_Percent - b.Women_Percent; });
+      } else if (options.sorted === 'desc') {
+        data.sort(function (a, b) { return b.Women_Percent - a.Women_Percent; });
+        // Note that this is just the opposite of the above.
+      }
+    }
+
+    // Filter Occupations to Major Group Types
+    if (options.filtered) {
+        if (options.filtered === 'Mgt') {
+            data = data.filter(function (d) {return d.Group === 'Mgt'})
+        } else if (options.filtered === 'Sr') {
+            data = data.filter(function (d) {return d.Group === 'Sr'})
+        } else if (options.filtered === 'Drs') {
+            data = data.filter(function (d) {return d.Group === 'Drs'})
+        } else if (options.filtered === 'Admin') {
+            data = data.filter(function (d) {return d.Group === 'Admin'})
+        }
+    }
+
+    // Select only the schools from the current page (slice)
+    if (options.page) {
```

Comment:
You've encountered one of Javascript's annoying quirks: 0 evaluates to false. This means that when we're on the first page (page 0), none of the stuff inside here runs and we see all the results.

This lead me to assume that you hadn't implemented pagination, but you have the most fully functional implementation of anyone in the class. Well done! (I have more comments on it below, as I'm wont to do.)